pytest.ini

```diff
@@ -1,3 +1,13 @@
 [pytest]
+addopts =
+    # show tests that (f)ailed, (E)rror, or (X)passed in the summary
+    -rfEX
+    # Make tracebacks shorter
+    --tb=native
+    # enable all warnings
+    -Wd
 markers =
     conda: conda required to run this test (deselect with '-m \"not conda\"')
+testpaths =
+    tests
```

Comment:
This is more of a QoL improvement. with this one can type `pytest` without specifying anything else to run these tests. LMK if you want me to revert this.